core/templates/dev/head/domain/statistics/statistics-domain.constants.ts

```diff
@@ -18,6 +18,7 @@
 
 export class StatisticsDomainConstants {
   public static LEARNER_ACTION_SCHEMA_LATEST_VERSION = 1;
+  public static ANSWER_DETAILS_IMPROVEMENT_CARD_TYPE = 'answerdetails';
```

Comment:
let's use hyphens, please rename to: `answer-details`

Reply:
Done